theano/scan_module/scan.py

```diff
@@ -600,9 +600,7 @@ def wrap_into_list(x):
         if init_out.get('taps', None) == [-1]:
 
             actual_arg = init_out['initial']
-            if not isinstance(actual_arg, tensor.Variable):
-                actual_arg = tensor.as_tensor_variable(actual_arg)
-            arg = safe_new(actual_arg)
+            arg = safe_new(init_out['initial'])
```

Comment:
This change isn't part of the problem. So we won't accept a revert of that part.